### ISSUE TITLE
Unit testing fails

### ISSUE THREAD

**Issue by qwertychouskie** [supertuxkart/stk-code#2929]
```
supertuxkart: stk-code/src/network/transport_address.cpp:110:
static void TransportAddress::unitTesting(): Assertion `t17.isLAN()' failed.
```

**Comment by Mrxx99** [comment]
@Benau: Since this issue is already fixed and the corresponding PR #3110 was closed because of this, this issue could be closed aswell.

**Comment by qwertychouskie** [comment]
Actually it was only fixed on the game protocol branch, that's fine though as it will be merged eventually.
